smoketest/webhook_test.go

```diff
@@ -0,0 +1,78 @@
+package smoketest
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/target/goalert/smoketest/harness"
+)
+
+type WebhookTestingAlert struct {
+	AlertID int
+	Type    string
+	Summary string
+	Details string
+}
+
+func TestWebhookAlert(t *testing.T) {
+	t.Parallel()
+
+	ch := make(chan WebhookTestingAlert)
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		var alert WebhookTestingAlert
+
+		data, err := ioutil.ReadAll(r.Body)
+		assert.Nil(t, err)
```

Comment:
Do a `require.Nil` so it panics/exits immediately, otherwise, it will hang until the test times out.